debian/changelog

```diff
@@ -1,3 +1,128 @@
+  * repo: add elementary to deb distros (#1637)
+
+  [ Leo Arias ]
+  * typo: replace occured with occurred
+  * python node: record installed node packages in manifest
+  * node plugin: record the yarn.lock file
+  * tests: fix the TEST_STORE environment variable
+  * tests: add integration tests for build snaps
+  * recording: record the machine information collected by uname
+  * tests: add unit tests for the ruby plugin
+  * recording: record the packages installed in the  machine
+  * tests: simplify a little the data in nodejs unit tests
+  * ci: use travis conditionals
+  * recording: record build-snaps installed during the pull
+  * tests: replace the first batch of demo tests with snapd integration tests
+  * rust plugin: record the Cargo.lock file
+  * rust plugin: record the versions of rustup, rustc and cargo
+  * tests: move ruby demo test to snapd integration suite (#1596)
+  * recording: record the snaps installed on the machine (#1567)
+  * style: use dedent for multiline strings (#1584)
+  * tests: refactor the fake snapd to not hardcode values (#1569)
+  * code style: remove the extra quotes in the dedent example (#1594)
+  * recording: do not crash when snapd is not installed (#1598)
+  * tests: fix the skip of snapd integration tests in armhf (#1595)
+  * tests: fix the duplicate plainbox test scenarios (#1608)
+  * tests: reenable the cleanbuild integration test (#1610)
+  * tests: remove the duplicate nodejs integration tests (#1609)
+  * tests: add /snap/bin to PATH in autopkgtests (#1603)
+  * tests: add the slow tag for ros snapd integration test (#1602)
+  * lxd: fix the unit test for the user id map (#1629)
+  * tests: allow to select a suite when running autopkgtests (#1630)
+  * tests: use the snapcraft snap for the integration tests (#1625)
+  * tests: move the plainbox test to the integration suite (#1642)
+  * lxd: fix the push in container builds (#1644)
+
+  [ Christian Dywan ]
+  * lxd: mount project folder via sshfs in case of a remote (#1302)
+  * lxd: Only pass target arch if specified explicitly
+  * repo: friendly, helpful error for unsupported distros (#1586)
+  * lxd: instructions for /etc/sub{u,g}id after failed start (#1553)
+  * lxd: pass SNAPCRAFT_PARTS_URI through into container (#1585)
+  * lxd: use SUDO_UID for ID mapping (#1588)
+  * lxd: don't inject local snaps on a different arch (#1577)
+  * lxd: don't re-inject the same snaps (#1568)
+  * lxd: split container classes into different files (#1627)
+  * cli: update parts cache in the container (#1546)
+  * lifecycle: clean after deleting container (#1587)
+
+  [ James Beedy ]
+  * ruby plugin: new plugin
+
+  [ Aleix Pol ]
+  * repo: return a proper value in DummyRepo
+  * common: do not fail over on empty or faulty lines in os-release
+
+  [ Jeff Dickey ]
+  * ci: install git in Dockerfile for '{version: git}' usage (#1575)
+
+  [ Michael Vogt ]
+  * meta: add and adapter property for apps
```

Comment:
There are a few lines that don't have PR numbers. Were those merged differently?